enterprise/frontend/src/metabase-enterprise/table-editing/settings.tsx

```diff
@@ -2,6 +2,12 @@ import type { Database } from "metabase-types/api";
 
 export const DATABASE_TABLE_EDITING_SETTING = "database-enable-table-editing";
 
+export const ENGINE_SUPPORTED_FOR_TABLE_EDITING = new Set([
+  "postgres",
+  "mysql",
+  "h2",
+]);
```

Comment:
We don't want to hard code this, we want to just rely on `hasFeature(database, "actions/data-editing").